docs/quickstart.md

```diff
@@ -125,7 +125,8 @@ Update the following variables specific to OCP local registry. Note that this is
 
 Update the following variables specific to OCP upgrade. The upgrade will be performed after a successful install of OCP.
 
- * `upgrade_image` : (Optional) OpenShift release image having higher and supported version. If set, OCP cluster will be upgraded to this image version. (e.g. `"quay.io/openshift-release-dev/ocp-release-nightly@sha256:552ed19a988fff336712a50..."`)
+ * `upgrade_version` : (Optional) OpenShift higher and supported version. If set, OCP cluster will be upgraded to this version. (e.g. `"4.5.4"`)
+ * `upgrade_channel` : (Optional) OpenShift channel having required upgrade version available for cluster upgrade. (stable-4.x, fast-4.x, candidate-4.x) e.g. stable-4.5
```

Comment:
`upgrade_channel` is required when `upgrade_version` is given right? as we are not setting any default to it. Please update in case.

Reply:
Internally upgrade_channel will be set to stable-4.x of installed cluster version. IMO we should keep it as optional.  I will add that note to it.

Reply:
It would be important to note what combinations are allowed .. 
eg. 
1. upgrade_version will upgrade to required version using the default channel
2. upgrade_version and upgrade_channel will upgrade to required version using the specified channel
3. upgrade_image will upgrade to required version using the image and doesn't need upgrade_version and upgrade_channel

Reply:
@bpradipt upgrade_version will upgrade to a required version only if that version is available in upgrade_channel whether the default one or set by the user.

The upgrade_image approach is removed as it is not the recommended way of doing an upgrade and may result in broken clusters most of the time. (Also it was not mentioned in official docs as well)

Reply:
Thanks @Prajyot-Parab .. Good to remove the upgrade_image param